docs/book/helpers/doctype.md

```diff
@@ -26,39 +26,58 @@ You can also specify a custom doctype as long as it is well-formed.
 The `Doctype` helper is a concrete implementation of the
 [Placeholder helper](placeholder.md).
 
-## Basic Usage
+## Configuration and Basic Usage
 
 You may specify the doctype at any time. However, helpers that depend on the
-doctype for their output will recognize it only after you have set it, so the
-easiest approach is to specify it in your bootstrap:
+doctype for their output will recognize it only after you have set it.
+There are multiple ways to set the doctype:
 
-```php
-use Laminas\View\Helper\Doctype;
+### Generic application configuration
 
-$doctypeHelper = new Doctype();
-$doctypeHelper->doctype('XHTML1_STRICT');
-```
-
-And then print it out on top of your layout script:
+`\Laminas\View\Helper\Service\DoctypeFactory` checks the application configuration, making it possible to
+define the doctype through your configuration (e.g. a `ConfigProvider`):
 
 ```php
-<?php echo $this->doctype() ?>
+return [
+    'view_helper_config' => [
+        'doctype' => 'HTML5',
+    ],
+];
 ```
 
-Within an application based off the [laminas-mvc-skeleton](https://github.com/laminas/laminas-mvc-skeleton),
-you can specify the doctype via configuration:
+### `laminas-mvc` configuration
+
+An application based off the [laminas-mvc-skeleton](https://github.com/laminas/laminas-mvc-skeleton)
+comes with its own factory in the `ViewHelperManager` and can be configured like this:
```

Comment:
The headline should be "Usage in a laminas-mvc Application" and the second should be used.

Reply:
The link to the "ViewManager" from the end of the page can be added here. [The last headline and paragraph](https://docs.laminas.dev/laminas-view/helpers/doctype/#laminas-mvc-view-manager) of the current page can be removed then.

Reply:
done!